src/button/ButtonGroup.tsx

```diff
@@ -43,7 +40,7 @@ export const ButtonGroup = forwardRefWithAs<ButtonGroupProps, "div">(
     const theme = useTheme();
     const buttonGroupStyles = cx(
       theme.buttonGroup.base,
-      isAttached ? theme.buttonGroup.attached : "",
+      isAttached ? theme.buttonGroup.attached : "lib:space-x-1",
```

Comment:
In order to eliminated the overriding issue within our components, we need to include classnames inside the components directly aswel itseems.